src/content/development/shipyard/advanced/_index.en.md

```diff
@@ -122,19 +122,28 @@ Flags of note:
   make deploy DEPLOY_ARGS='--deploytool operator --deploytool_submariner_args "--cable-driver wireguard"'
   ```
 
+As shown above, arguments can be passed directly to the Broker and Submariner.
+The deploy script also has some additional flags that can be set and the script will set the Broker and Submariner arguments properly for you (see the flags defined in [deploy.sh](https://github.com/submariner-io/shipyard/blob/devel/scripts/shared/deploy.sh) for the full list):
```

Comment:
Took me a minute to figure out what you meant here, this might be more clear:

```suggestion
The deploy script also has flags that group common options together for easier UX. For example, the [`service_discovery` flag in `deploy.sh`](https://github.com/submariner-io/shipyard/blob/devel/scripts/shared/deploy.sh) will handle the `--components service-discovery,connectivity` flags mentioned above.
```

Reply:
Nice edit in the latest version, thanks @Billy99!